linux_os/guide/system/logging/rsyslog_sending_messages/rsyslog_remote_tls/rule.yml

```diff
@@ -0,0 +1,32 @@
+documentation_complete: true
+
+prodtype: rhel8,fedora,ol8
+
+title: 'Configure TLS for rsyslog remote logging'
+
+description: |-
+    Configure <tt>rsyslog</tt> to use Transport Layer
+    Security (TLS) support for logging to remote server.
```

Comment:
We usually describe also which option needs to be configured and in which configuration file this option can be found.

Reply:
Rule descriptions expanded -> c49f0b5b32ef05db833004a77a8183e065e1793b.